qiskit/transpiler/passes/layout/disjoint_utils.py

```diff
@@ -141,6 +141,7 @@ def combine_barriers(dag: DAGCircuit, retain_uuid: bool = True):
 def require_layout_isolated_to_component(
     dag: DAGCircuit, components_source: Union[Target, CouplingMap]
 ) -> bool:
```

Comment:
wrong annotation or should return value?

Reply:
Good catch. Based on the callers, it should be `None`. [Here](https://github.com/Qiskit/qiskit-terra/pull/8266/files#r1197538178) is an extra documentation for it :)